src/dev/impl/DevToys/ViewModels/Tools/Graphic/ImageConverter/ImageConverterToolViewModel.cs

```diff
@@ -0,0 +1,249 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Composition;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using DevToys.Api.Tools;
+using DevToys.Core.Threading;
+using DevToys.Helpers;
+using DevToys.Shared.Core;
+using DevToys.Views.Tools.ImageConverter;
+using Microsoft.Toolkit.Mvvm.ComponentModel;
+using Microsoft.Toolkit.Mvvm.Input;
+using Windows.ApplicationModel.DataTransfer;
+using Windows.Graphics.Imaging;
+using Windows.Storage;
+using Windows.Storage.Pickers;
+using Windows.Storage.Streams;
+using Windows.UI.Xaml;
+
+namespace DevToys.ViewModels.Tools.ImageConverter
+{
+    [Export(typeof(ImageConverterToolViewModel))]
+    public sealed class ImageConverterToolViewModel : ObservableRecipient, IToolViewModel, IDisposable
+    {
+        private static readonly string[] SupportedFileExtensions = new[] { ".png", ".jpg", ".jpeg", ".jfif", ".pjpeg", ".pjp", ".jxr", ".bmp", ".tiff", ".tif", ".heif", ".gif" };
```

Comment:
I see that GIF files are supported. I'm afraid this will confuse the user as most of GIF are animated and that the tool would convert just the first frame of the animation. I'd suggest removing the GIF support for now and wait for users feedback to see if people actually want it.

Additionally, AFAIK, jfif, pjpeg, pjp, jxr and heif aren't the most common used image format in development (but I can be wrong). 
I'd suggest either sticking to PNG, JPG, JPEG and BMP or updating `InvalidSelectedFiles` to indicate all these formats. But to be fair, unless someone convinces me otherwise, I think it makes sense to keep it simple and just support PNG, JPG, JPEG and BMP for now. 
What do you think?

Reply:
I added GIF feature just because I was using all the codecs present in the library, but yes, that makes sense.

From my point of view, maybe someone at some point can find usefull the HEIF/JPEG XR/TIFF options. We can limit using ```SupportedFileExtensions``` variable the extensions allowed (just let me know which extensions should be delisted), but keep those encoding options may be a differentiating factor compared to other tools/online options.

Reply:
I see :)
For now, let's try to support:
- PNG
- JPG
- JPEG
And let's see if users are asking for more.

Reply:
I have already added only support for PNG, JPEG and BMP (as you included it in the previous comment) formats

Reply:
> maybe someone at some point can find usefull the HEIF/JPEG XR/TIFF options

As someone who works in technical services, I regularly get end users sending me photos as HEIF files, which get held in our mail filter (they are classified under "media files" along with HEVC/MP4 and I can't change this). Our teams have tools they use to convert these to JPG and re-attach to the email, but there's nothing standardized (just various online/portable apps). I would definitely use this feature for these files if it were available.